templates/src/index.js

```diff
@@ -0,0 +1,25 @@
+import { getOptions } from 'loader-utils';
+import validateOptions from 'schema-utils';
+import schema from './options.json';
+
+export const raw = true;
+
+export default function loader(source) {
+  const { version, webpack } = this;
+
+  const options = getOptions(this) || {};
+
+  validateOptions(schema, options, 'Loader');
+
+  const newSource = `
```

Comment:
`newSource` => `result` ?

```js
export default function loader (src, map, meta) {
  ...
  // Sync Loader
  return result;

  // Async Loader
  const cb = this.async();

  cb(err, result, map, meta);
  // Avoid 'runaway' {Promise}
  return null;
}
```

Reply:
We can do this anyway you want. I look at this template in particular as something that is going be deleted right out of the gate when someone actually starts creating a new loader.

It's super basic on purpose. The existing `index.js` exists almost entirely to ensure that the test setup works.

Reply:
Yeah... we should not overcomplicating this starter file (e.g my comment mixing sync/async is definitely already to verbose), only a few links to the Loader Types/API and the Writing Guide on `webpack.js.org` maybe a good idea, I will think about this and checkout the docs, but non-blocking in anyway imho and minor important for this PR (the current code is perfectly fine for getting started)

ℹ️ This when the CLI interface is ready should also switch between a loader and a plugin template then, but you're aware of that I guess 😛